python_examples/sim_speeds/Readme.md

```diff
@@ -0,0 +1,6 @@
+The 4 csv files in this folder contain data on the speed of our code before we sped it up. 
```

Comment:
This is a really nice description, how difficult would it be to get the actual commits from our github history where these were run from? If can get them quickly that would be the most reliable information to save about them.